### PR TITLE
Added newsletter

### LINKED ISSUE #263: Adding a newsletter component
### **Feature request**:

Adding a component for 'Signup for newsletter'

### **Describe the solution you'd like**:

I would like to design and implement a react component for newsletter subscription for this website

### **Describe how this feature will be useful to our readers**:

Users can subscribe to get opportunity alerts at the right time

<hr/>

### PR DESCRIPTION
**Pull request by sandra9711**
# Description

Added a component for newsletter with link in products part of footer

Fixed #263 

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] I have pulled the latest changes in the repository
- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by vaishali614** [comment]
@sandra9711 Apologies for the delay. 
We don't want a separate page for subscribing to our newsletter, instead you can just add an input field with a button to Subscribe, like [here](https://seekho.ai/academy/product-management).
Also, Newsletter is not a Product, so remove it from there and simply add this input field above "Get in Touch" section.

**Comment by sandra9711** [comment]
@vaishali614 
Made all changes mentioned

**Comment by vaishali614** [comment]
> @vaishali614
> Made all changes mentioned

It's not in center, please keep it aligned with other components. Make sure to check for all screen-sizes. Other than that, it looks good to go.
Screenshot (158)

**Comment by sandra9711** [comment]
@vaishali614 

I corrected the alignment as in this picture

oc3

But it looks like this in the deploy preview

oc4

**Review by vaishali614 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -162,6 +163,7 @@ function App() {
               </div>
             )}
           />
+          <Route path="/newsletter" component={Newsletter} strict exact/>
```
Comment (vaishali614) [inline src/App.js]: This is not required now

File: src/CSS/Newsletter.module.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
+  
+
```
Comment (vaishali614) [inline src/CSS/Newsletter.module.css]: Please remove unnecessary code from this file

File: src/Components/HomePage/Sections/Navbar.js [hunk 3]
```diff
@@ -28,6 +28,9 @@ function navbar() {
           <Link smooth={true} offset={-90} duration={1000} to="faqs">
             <a className={styles.NavLinks}>FAQs</a>
           </Link>
+          <Link smooth={true} offset={-90} duration={1000} to="faqs">
```
Comment (vaishali614) [inline src/Components/HomePage/Sections/Navbar.js]: Remove it from navbar too

**Review by vaishali614 — CHANGES_REQUESTED** [review]

File: src/Assets/subscribe.svg [hunk 4]
```diff
@@ -0,0 +1,74 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0px" y="0px" viewBox="0 0 1080 1080" enable-background="new 0 0 1080 1080" xml:space="preserve">
```
Comment (vaishali614) [inline src/Assets/subscribe.svg]: Why is this svg file added? I don't see it getting used anywhere. Please remove this if it's unnecessary.

**Review by vaishali614 — CHANGES_REQUESTED** [review]

**Review by vaishali614 — APPROVED** [review]

**Pull request merged**